openapi/components/schemas/Invoices/TaxTypes/rebilly-taxjar.yaml

```
@@ -1,12 +1,15 @@
-description: Taxjar-based taxes.
 allOf:
   - $ref: ../InvoiceTax.yaml
   - type: object
+    required:
+      - calculator
     properties:
+      calculator:
+        description: Type of tax calculator.
+        type: string
       items:
         type: array
-        description: |-
-          Taxes that are applied to this invoice and calculated based on the related product's `taxCategoryId`.
+        description: Taxes that are applied to this invoice and calculated based on the related product's `taxCategoryId`.
```

issue: where possible, avoid [possessive apostrophes](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#possessive-apostrophes)
```suggestion
        description: Taxes that are applied to this invoice and calculated based on the `taxCategoryId` of the related product .
```